generate_weekly_newsletter_json.py

```diff
@@ -20,7 +20,8 @@
     elastic_search = ElasticSearchClient()
     dev_urls = [
         ["https://lists.linuxfoundation.org/pipermail/bitcoin-dev/",
-         "https://gnusha.org/pi/bitcoindev/"],
+         "https://gnusha.org/pi/bitcoindev/",
```

Comment:
What is the reason that we are keeping `https://gnusha.org/pi/bitcoindev` here? 
This also extends to `https://lists.linuxfoundation.org/pipermail/bitcoin-dev/` which is in the above line.

Reply:
Keeping `https://gnusha.org/pi/bitcoindev` only as a fallback approach. 

And for `https://lists.linuxfoundation.org/pipermail/bitcoin-dev/`, if we are sure that there won't be any new thread there, we can remove it.

Should I remove them both?

Reply:
> Keeping https://gnusha.org/pi/bitcoindev only as a fallback approach.

what fallback means in this case?

When we scrape the mailing list we scrape just `https://mailing-list.bitcoindevs.xyz/bitcoindev/`, right?

Reply:
The workflow is like this - 
In Scraper Repo:
1. Scrape the docs from `https://gnusha.org/pi/bitcoindev` and insert them into ES
2. While inserting these docs the change the `domain` URL from `https://gnusha.org/pi/bitcoindev` to `https://mailing-list.bitcoindevs.xyz/bitcoindev`  

In Mailing List Summaries Repo: 
3. Fetch the documents for respective URLs provided in the list and perform the operations.

Reply:
okay, thanks for the clarification. Can you remind me why we are doing this "domain injection"? Why not scrape directly from `https://mailing-list.bitcoindevs.xyz/bitcoindev`

Reply:
Basically `https://gnusha.org/pi/bitcoindev` is our data source, and `https://mailing-list.bitcoindevs.xyz/bitcoindev` is our custom domain where we are managing [Bitcoin TLDR](https://tldr.bitcoinsearch.xyz/)

For more info refer: [this chat](https://discord.com/channels/1189471179967315968/1255834702972194827/1255858797923270796)